firebase-config/src/main/java/com/google/firebase/remoteconfig/RemoteConfigConstants.java

```diff
@@ -59,6 +60,7 @@
     String PLATFORM_VERSION = "platformVersion";
     String TIME_ZONE = "timeZone";
     String APP_VERSION = "appVersion";
+    String APP_BUILD = "appBuild";
```

Comment:
Looks like a consistent pattern that the [iOS constants use underscores](https://github.com/firebase/firebase-ios-sdk/blob/f3bb73fc8950f4cc6041d3e66ab6b40ee05cc698/FirebaseRemoteConfig/Sources/RCNDevice.m#L29) and Android uses camel case